test/integration/image-optimizer/test/disable-write-to-cache-dir.test.ts

```diff
@@ -0,0 +1,13 @@
+import { join } from 'path'
+import { setupTests } from './util'
+
+const appDir = join(__dirname, '../app')
+const imagesDir = join(appDir, '.next', 'cache', 'images')
+
+describe('with isrFlushToDisk config', () => {
```

Comment:
```suggestion
describe('with isrFlushToDisk: false config', () => {
```

Let's update the existing tests to remove this new property and so that this test is the only one that uses `false`